zephyr/sensor_service.c

```diff
@@ -11,6 +11,7 @@
 #include <zephyr.h>
 #include <sensor.h>
 #include <shell/shell.h>
+#include <logging/sys_log.h>
```

Comment:
now that I am checking the zephyr docs I see there're actually two (actually three) log systems:

- system log (CONFIG_SYS_LOG) https://docs.zephyrproject.org/1.12.0/subsystems/logging/system_log.html
- logger (CONFIG_LOG) https://docs.zephyrproject.org/latest/subsystems/logging/logger.html

logger seems to be better suited for our general logging, could you take a look at it? There're some examples in `samples/subsys/logging`.

Reply:
it's up to you; I usually squash the commits and force push to the branch, but opening a new PR is also ok :)

Reply:
#137